### DISCUSSION TITLE
Demo site is up again!

### DISCUSSION THREAD

**Discussion by edwardch** [dfaruque/Serenity.Extra#62] [Show and tell]
Hey all,

I have put the demo site back online here: [https://serenity-extra.azurewebsites.net](https://serenity-extra.azurewebsites.net).

There is no CI/CD in place and I will check periodically for updates before building and deploying, otherwise @dfaruque please let me know if you want stuff deployed and I'll try get to it.

The site is on Azure App Service free tier, so initial load might need some warmup time.

HTH!

Edward

**Comment by dfaruque** [comment]
Thanks a lot.

**Reply by edwardch** [reply]
Hey @dfaruque, 

FYI the latest changes have been deployed to the [demo site](https://serenity-extra.azurewebsites.net/).

It would be great for the demo to use the Serenity 5+ version but when I run it locally I get a few errors.

I understand you are busy with office work, so when you get a chance to update the SerExtraNet5.Web project please let me know.

Cheers!
Edward

**Reply by dfaruque** [reply]
Thanks, man.

**Reply by edwardch** [reply]
Hey @dfaruque, 

I have unfortunately had to take the demo site off our Azure account after we had to audit the resources used in our organisation.

Since it is cross platform now maybe someone would be willing to host on a DigitalOcean droplet, or similar. 

Sorry for inconvenience, I hope it was useful... I see it is on your Readme page, might need to remove that now.

Cheers,
Edward

**Reply by dfaruque** [reply]
It is okay. Thank for the info.
